cluster/images/crossplane/Makefile

```diff
@@ -1,31 +1,38 @@
 # ====================================================================================
 # Setup Project
 
-PLATFORMS := linux_amd64 linux_arm64 linux_arm linux_ppc64le
 include ../../../build/makelib/common.mk
 
 # ====================================================================================
 #  Options
-IMAGE = $(BUILD_REGISTRY)/crossplane-$(ARCH)
+
 OSBASEIMAGE = gcr.io/distroless/static@sha256:d2b0ec3141031720cf5eedef3493b8e129bc91935a43b50562fbe5429878d96b
-include ../../../build/makelib/image.mk
+include ../../../build/makelib/imagelight.mk
 
 # ====================================================================================
 # Targets
 
-ifeq ($(PLATFORM),$(filter $(PLATFORM),darwin_amd64 darwin_arm64 windows_amd64)) 
-$(info Skipping image build for $(PLATFORM))
-img.build:
-else
 img.build:
 	@$(INFO) docker build $(IMAGE)
+	@$(MAKE) BUILD_ARGS="--load" img.build.shared
+	@$(OK) docker build $(IMAGE)
+
+img.publish:
+	@$(INFO) docker publish $(IMAGE)
+	@$(MAKE) BUILD_ARGS="--push" img.build.shared
+	@$(OK) docker publish $(IMAGE)
+
+img.build.shared:
 	@cp Dockerfile $(IMAGE_TEMP_DIR) || $(FAIL)
 	@cp $(OUTPUT_DIR)/bin/$(OS)_$(ARCH)/crossplane $(IMAGE_TEMP_DIR) || $(FAIL)
 	@cp -a ../../../cluster/crds $(IMAGE_TEMP_DIR) || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && $(SED_CMD) 's|BASEIMAGE|$(OSBASEIMAGE)|g' Dockerfile || $(FAIL)
-	@docker build $(BUILD_ARGS) \
-		--platform linux/$(ARCH) \
+	@docker buildx build $(BUILD_ARGS) \
+		--platform $(IMAGE_PLATFORMS) \
```

Comment:
@hasheddan have we verified that you're getting the expected archs given this multi-platform step is using a multi-arch base image? My hope is that `buildx` is handling this better than `docker build`. If not, it probably wouldn't hurt to do a quick `inspect` on the resulting images that come back from a `docker pull --platform x`.

Reply:
@tnthornton we are getting the correct arch (checked via inspect, see images referenced in PR body), but we are going to need to update the publish step because we aren’t using the correct binaries because we build them outside and copy them in. I’m going to adjust the build process to accommodate.

Reply:
Awesome! Sorry I missed those links the first time through.